pkg/gencert/gencert.go

```diff
@@ -0,0 +1,179 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gencert
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	certificates "k8s.io/api/certificates/v1beta1"
```

Comment:
Typically you separate these things into three groups:

* built-in imports (like time and fmt)
* local repo imports (like k8s.io/test-infra/foo/bar)
* foreign repo imports (like k8s.io/api/core/v1, github.com/random-org/random-repo)

Reply:
Thanks, I was not aware of this convention. I have updated my editor accordingly.